src/applications/personalization/dashboard/tests/actions/claims.unit.spec.jsx

```diff
@@ -0,0 +1,164 @@
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+import {
+  mockFetch,
+  setFetchJSONFailure,
+  setFetchJSONResponse,
+} from '@department-of-veterans-affairs/platform-testing/helpers';
+
+import { createClaimsSuccess } from '../../mocks/claims';
+import { createAppealsSuccess } from '../../mocks/appeals-success';
+import { getAppealsV2, getClaims } from '../../actions/claims';
+
+import {
+  FETCH_CLAIMS_PENDING,
+  FETCH_CLAIMS_ERROR,
+  FETCH_CLAIMS_SUCCESS,
+} from '../../utils/claims-helpers';
+import {
+  FETCH_APPEALS_PENDING,
+  FETCH_APPEALS_SUCCESS,
+  USER_FORBIDDEN_ERROR,
+  RECORD_NOT_FOUND_ERROR,
+  VALIDATION_ERROR,
+  BACKEND_SERVICE_ERROR,
+  FETCH_APPEALS_ERROR,
+} from '../../utils/appeals-helpers';
+
+describe('/actions/claims', () => {
+  describe('getAppealsV2', () => {
+    let dispatchSpy;
+    let oldDataLayer;
+    beforeEach(() => {
+      mockFetch();
+      oldDataLayer = global.window.dataLayer;
+      global.window.dataLayer = [];
+      dispatchSpy = sinon.spy();
+    });
+    afterEach(() => {
+      global.window.dataLayer = oldDataLayer;
+    });
```

Comment:
So this particular set of tests for the `getAppealsV2` action doesn't use the `dataLayer` at all during the action flow. You can remove all the beforeEach / afterEach stuff that pertains to it and it will still pass.

Should this action have analytics events attached to it? Maybe so... probably worth bringing up to AJ for discussion.

Reply:
🤦 thank you for catching that. i'll remove it for now.

i'll also reach out to AJ about tracking API calls to the `/appeals` endpoint